e2e/test/scenarios/dashboard/actions-on-dashboards.cy.spec.js

```diff
@@ -581,6 +585,118 @@ const MODEL_NAME = "Test Action Model";
           });
         });
       });
+
+      describe("editing action before executing it", () => {
```

Comment:
It seems that we only test here that we can change the action name and execute the modal, which is great. But can we also test what happens if we change the action parameters? What happens to the existing form fields? You could probably just add 1 extra e2e test and combine multiple cases into one.